website/source/docs/agent/http/agent.html.markdown

```diff
@@ -231,14 +245,26 @@ query parameter causes the agent to attempt to join using the WAN pool.
 
 The return code is 200 on success.
 
+### <a name="agent_leave"></a> /v1/agent/leave
+
+Added in Consul 0.7.2, this endpoint is hit with a PUT and is used to trigger a
+graceful leave and shutdown of the agent. It is used to ensure other nodes see the
+agent as "left" instead of "failed". Nodes that leave will not attempt to re-join
+the cluster on restarting with a snapshot.
+
+For nodes in server mode, the node is removed from the Raft peer set in a graceful
```

Comment:
This isn't true any more after Consul 0.7.0 so we should change this, and reference the `leave_on_terminate` config option. Did this get copied over from some place? We should fix the source  as well.